components/schemes/Typography/src/MDCTypographyScheme.h

```diff
@@ -85,6 +85,14 @@
  */
 @property(nonatomic, nonnull, readonly, copy) UIFont *overline;
 
+@optional
```

Comment:
Please remove this @optional marker.  My understanding is that subsystem schemes do not have optional properties. Instead, all clients should conform to this API and simply return `NO` if they do not support Material Dynamic Type in the fonts.

Reply:
Won't that break everyone with an existing implementation?

Reply:
Yes, that is expected and one of the reasons we have decided so far to avoid `@optional` properties.  Clients can see the breakage and decide how to proceed.  For internal clients, we can add the property as a method that returns `NO`. For external clients the release will be a Major bump with (simple) migration steps.

Reply:
Done.